module/VuFind/src/VuFind/Db/Entity/ResourceTags.php

```diff
@@ -38,63 +38,90 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
  *
- * @ORM\Table(name="resource_tags", indexes={@ORM\Index(name="list_id", columns={"list_id"}), @ORM\Index(name="resource_id", columns={"resource_id"}), @ORM\Index(name="tag_id", columns={"tag_id"}), @ORM\Index(name="user_id", columns={"user_id"})})
+ * @ORM\Table(name="resource_tags",
+ * indexes={@ORM\Index(name="list_id", columns={"list_id"}),
+ * @ORM\Index(name="resource_id",   columns={"resource_id"}),
```

Comment:
More foreign key index definitions -- I'm pretty sure that these were autogenerated by the initial entity creation script, so I'm not sure why there's an inconsistency here. As with other things, I don't have a strong opinion about whether these should or should not be present, but we should make sure we're consistent across all tables with foreign keys.